### DISCUSSION TITLE
Tailwind css not working on linux mint

### DISCUSSION THREAD

**Discussion by xNyfPtx** [tailwindlabs/tailwindcss#13943] [Help]
System info: 
![image](https://github.com/tailwindlabs/tailwindcss/assets/155147963/b36fa3e3-2ee1-4de5-a2aa-10656e3d4027)

Note: all of the comand that i type in the termianl are all on the home direcotry and without the use of vscode inbuilt terminal but rather the provided terminal app on linxh mint

what i didi first:  i ran npm install -D tailwindcss on the terminal and got this error or maybe not 
![image](https://github.com/tailwindlabs/tailwindcss/assets/155147963/5fbb4561-b51c-4525-8905-c637339701d8)

second: i ran  npx tailwindcss init in the terminal and maybe this ian  error 
![image](https://github.com/tailwindlabs/tailwindcss/assets/155147963/8f6e659e-1de5-4dc9-8c04-3efa5c203145)

how do you fix this

**Comment by sceee** [comment]
In your second screenshot, `npm` tells you that you have node 12 installed which is [end of life already since 2022](https://github.com/nodejs/Release?tab=readme-ov-file#end-of-life-releases) and several packages require a `node` version >= 14 or 16.

=> Try updating `node` on your system to [CURRENT or LTS](https://github.com/nodejs/Release?tab=readme-ov-file#release-schedule).

**Reply by xNyfPtx** [reply]
![image](https://github.com/tailwindlabs/tailwindcss/assets/155147963/1af7799f-4538-4eb9-b40b-ea24a85f6192)
updated it, is this correct?

**Reply by sceee** [reply]
Looks better. Did it work now?
